packages/perseus/src/widgets/graded-group/graded-group.test.ts

```diff
@@ -329,4 +335,82 @@ describe("graded-group", () => {
             ).not.toBeInTheDocument();
         });
     });
+
+    /**
+     * GradedGroup and GradedGroupSet are only for articles,
+     * so we'll test them in that environment
+     */
+    describe("in ArticleRenderer", () => {
+        function generateArticle(): PerseusArticle {
+            return [
+                {
+                    content: "[[☃ graded-group 1]]",
+                    widgets: {
+                        "graded-group 1": {
+                            type: "graded-group",
+                            options: {
+                                title: "Test title",
+                                content: "[[☃ dropdown 1]]",
+                                widgets: {
+                                    "dropdown 1": {
+                                        type: "dropdown",
+                                        options: {
+                                            choices: [
+                                                {
+                                                    content: "Wrong answer",
+                                                    correct: false,
+                                                },
+                                                {
+                                                    content: "Right answer",
+                                                    correct: true,
+                                                },
+                                            ],
+                                            placeholder: "Choose an answer",
+                                            static: false,
+                                        },
+                                    },
+                                },
+                                images: {},
+                            },
+                        },
+                    },
+                    images: {},
+                },
+            ];
+        }
+
+        it("can be answered correctly", async () => {
+            renderArticle(generateArticle());
+
+            // select an option
+            const dropdown = screen.getByRole("combobox");
+            await userEvent.click(dropdown);
+            await userEvent.click(screen.getByText("Right answer"));
+
+            // make sure the option was selected properly
+            expect(dropdown).toHaveTextContent("Right answer");
+
+            await userEvent.click(screen.getByRole("button", {name: "Check"}));
```

Comment:
FWIW: not sure this adds clarity, or not, but I think you could use the `checkAnswer()` function here too.